File: 1.java
//создаем список
import java.util.ArrayList;

public class Main {

  public static void main(String[] args) {

    ArrayList<String> stack = new ArrayList<String>();

// доюавляем элементы в список
    stack.add("John");
    stack.add("Jane");
    stack.add("Doe");

//выводим список
    System.out.println(stack);
    // [John, Jane, Doe]

  }
}

File: 1.cpp
#include <iostream>
#include<stack>
using namespace std;

void printL(stack<int>& l) {
    for (auto i : l)
        cout << i << " ";
    cout << '\n';
}

int main() {
    
    // создаем пустой список
    stack<int> l1;

    // сощздаем список из списка инициализаторов
    stack<int> l2 = {1, 3, 4, 2, 5};

    //создаем список с заданным размером
    stack<int> l3(5, 9);
    
    printL(l1);
    printL(l2);
    printL(l3);
    return 0;
}
